GCMtools/plotting.py

```diff
@@ -49,13 +57,115 @@ def isobaric_slice(self, ds, var_key, p):
         p : float
             Pressure level for the isobaric slice to be plotted, expressed in
             the units specified in the init-method.
+        time : int, optional
+            Timestamp that should be plotted. By default, the last time is
+            selected.
+        ax : matplotlib.axes.Axes, optional
+            The axis on which you want your plot to appear.
+        plot_windvectors : boolean, optional
+            If True (default), plot the horizontal wind vectors over the colour
+            mesh.
+        wind_kwargs : dict, optional
+            Additional keywords for the method plot_horizontal_wind.
+        cbar_kwargs : dict, optional
+            Additional keywords for the colorbar.
+        fs_labels : int, optional
+            Font size of the axis labels, 14 by default.
+        fs_ticks : int, optional
+            Font size of the tick labels, 11 by default.
+        title : str, optional
+            Title for the isobaric slice plot. By default, the selected pressure
+            and time stamp of the slice are displayed.
+        xlabel : str, optional
+            X-axis label, longitude by default.
+        ylabel : str, optional
+            Y-axis label, latitude by default.
         """
+        if ax is None:
+            fig= plt.figure()
+            ax = plt.gca()
+        if wind_kwargs is None:
+            wind_kwargs = {}
+        if cbar_kwargs is None:
+            cbar_kwargs = {}
+
+        # retrieve default units
         p_unit = ds.attrs.get('p_unit')
-        print('We are working in units: ' + p_unit)
-        # TODO: implement
-        pass
+        time_unit = ds.attrs.get('time_unit')
+
+        # By default, the plotted...
+        #   - pressure is the nearest to p
+        #   - iteration is the last one
+        this_p = ds.Z.sel(Z=p, method="nearest").values
```

Comment:
```Suggestion
        this_p = ds.Z.interp(Z=p).values
```

Reply:
I agree that interpolation could be very useful, but I don't think it should be done in an automatic/blackboxy way, because that can often lead to issues. E.g. not all quantities benefit from linear interpolations, especially not on a pressure grid. 

Would it be a solution to implement different lookup-methods (exact, nearest, interpolation)?

Reply:
agree! Is that easy to do?

Reply:
Should be easy. Most of it is done via xarray already.